tls/tls_srv.c

```diff
@@ -1121,6 +1142,10 @@ ttls_parse_client_hello(TlsCtx *tls, unsigned char *buf, size_t len,
 	 */
 	ttls_process_session_ticket(tls);
 
+	/* JA5t computation */
+	tls->sess.ja5t.is_abbreviated = tls->hs->resume;
+	tls->sess.ja5t.alpn = tls->alpn_chosen->id;;
```

Comment:
Not exactly: this is choosen ALPN, but we need ALPN, sent by a client, so it seems `ttls_parse_alpn_ext()` is a better place. E.g. if we end up with HTTP/2, we still need to fingerprint `http/1.1,h2`. Ideally, if we can handle the difference between `http/1.1,h2` and `h2,http/1.1`, but not necessary - if it causes overhead, we'll be just fine with setting bits for h2 or http/1.1

Reply:
done